src/test/java/de/unistuttgart/iste/gits/content_service/api/mutation/MutationDeleteContentTest.java

```diff
@@ -85,8 +86,61 @@ void testDeleteExistingContent(GraphQlTester graphQlTester) {
         // Tags are not deleted (yet)
         assertThat(tagRepository.count(), is(2L));
 
-        verify(topicPublisher, times(1))
```

Comment:
Der ist vermutlich verbuggt, weil man den in @AfterEach resetten muss. 
Aber wie du sagst, wird ja im Service getestet.